D/DASKR/Compat.toml

```diff
@@ -1,15 +1,16 @@
 [2]
 Compat = "0.17-2"
-Reexport = "0"
 
 ["2-2.2"]
 julia = "0.7-1"
 
 ["2-2.3"]
 DiffEqBase = "3-5"
+Reexport = "0.0.0 - 0.2"
 
 ["2.3-2"]
 julia = "1"
 
 ["2.4-2"]
 DiffEqBase = "5"
+Reexport = "0"
```

Comment:
What about this?

Reply:
Yeah. That’s because this PR was run with the “exclude the latest version of each package” option.

If we would want to also cap the latest version of each package, we would just need to run it with the “cap the latest version of each package“ option: https://github.com/bcbi/RetroCap.jl#run-on-all-repos-in-a-registry

Reply:
Why would you not want to run it for the latest version?

Reply:
Yeah, that seems odd. This is the only library I forgot to put a tag in for, so therefore it's the only one in the future that won't have an upper bound? I don't quite get it. But it seems this is the only one- so it can be spot fixed.

Reply:
So here is the backstory. The original motivation for RetroCap was an idea by Stefan in which we would allow package authors to not have upper bounds on the latest versions of their packages, and then every time they released a new version, we would go back and retroactively add caps to all earlier versions.

So that’s why RetroCap was originally set up to add caps for all versions of a package except the latest version.

Anyway, the new version of RetroCap lets you pick whether or not you want to add caps for the latest version: https://github.com/bcbi/RetroCap.jl/blob/master/README.md#run-on-all-repos-in-a-registry